src/client/components/timeseries-visualization-controls/visualization-controls.tsx

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright 2017-2022 Allegro.pl
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from "react";
+import { classNames } from "../../utils/dom/dom";
+import {
+  VisualizationControls,
+  VisualizationControlsBaseProps
+} from "../../views/cube-view/center-panel/center-panel";
+import { SplitTileBaseProps } from "../split-tile/split-tile";
+import { SplitTilesRow, SplitTilesRowBaseProps } from "../split-tile/split-tiles-row";
+import { SvgIcon } from "../svg-icon/svg-icon";
+import { WithRef } from "../with-ref/with-ref";
+import { TimeSeriesSplitMenu } from "./split-menu";
+
+const TimeSeriesSplitTile: React.FunctionComponent<SplitTileBaseProps> = props => {
```

Comment:
looks very similar to https://github.com/allegro/turnilo/blob/master/src/client/visualizations/grid/grid-split-tile.tsx#L26

Main difference now is "when we can open menu". I'm not sure if we can express that as a simple config/flag for this component. I'd like to not overenginner this.

Reply:
Ok, it is exactly the same as https://github.com/allegro/turnilo/blob/master/src/client/components/split-tile/split-tile.tsx#L53 ... so now it is much simpler.